ci-jobs/templates/android-e2e-template.yml

```diff
@@ -40,6 +40,8 @@ jobs:
       displayName: Prebuild the server
     - script: npx mocha --timeout 300000 --reporter mocha-multi-reporters --reporter-options config-file=./ci-jobs/mocha-config.json --recursive build/test/functional/driver-e2e-specs.js -g @skip-ci -i --exit
       displayName: Run tests
+    - script: npx mocha --timeout 300000 --reporter mocha-multi-reporters --reporter-options config-file=./ci-jobs/mocha-config.json --recursive build/test/functional/commands/jetpack-compose-attributes-e2e-specs.js -g @skip-ci -i --exit
```

Comment:
I assume this test could be run in parallel with the previous one

Reply:
mm, let me try

Reply:
Meaning... create multiple emulators and run tests with them?
Then yes, i think we can with some device handling changes.
I'd like to do such changes in another pr. (this pr updates only `jetpack-compose-attributes-e2e-specs.js` methods for webdriverio)